tests/integration/component.go

```diff
@@ -51,8 +51,8 @@ func componentTests(args ...string) {
 			Expect(output).NotTo(ContainSubstring("Specify output format, supported format: json"))
 		})
 
-		It("Help for odo create should contain machine output", func() {
-			output := helper.CmdShouldPass("odo", "create", "--help")
+		It("Help for odo project list should contain machine output", func() {
+			output := helper.CmdShouldPass("odo", "project", "list", "--help")
```

Comment:
since this is testing `project list`, I would suggest moving this to `cmd_project_test.go`

Reply:
Got it :), moving!

Reply:
It's not moved yet :D

Reply:
Done! Thanks